### DISCUSSION TITLE
Download From User Profile

### DISCUSSION THREAD

**Discussion by tamon504** [Tairraos/TikDown#13] [Ideas]
Is there any way to download all video from spesific users on douyin.? rather than copy and paste the links one by one.?

many thanks for great code

**Comment by Tairraos** [comment]
If you want to have this feature right now, there is an free app provided this feature: 4K Tokkit(50 videos per day, but you can change system calender to get more credit). 

Or you can turn on the watch clipboard feature and copy each video share string manually. When the feature is on, app will try to parse clipboard content after clipboard changed.

I will do some research how to download by a user name after next month, since I am busy on another hardware project now.

**Comment by viking1304** [comment]
> If you want to have this feature right now, there is an free app provided this feature: 4K Tokkit(50 videos per day, but you can change system calender to get more credit).

Tokkit doesn't work with douyin at all. It only works with "international" TikTok. That's the reason I prefer your app. I made brew formula yesterday, but I need to finish zap stanza in order to send a pull request to homebrew repo.

> I will do some research how to download by a user name after next month, since I am busy on another hardware project now.

Even if we have to wait for a few months, you deserve our patience. Thank you for the great work!
